.github/workflows/ci.yml

```diff
@@ -0,0 +1,288 @@
+name: Rust
+on:
+  push:
+    branches:
+      - master
+      - ci # TODO: remove later
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Setup Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
```

Comment:
is there any benefit from running rustfmt on nightly?

Reply:
The current rustfmt config have an options which supported by nightly only.
I would like prefer remove config and use default formatting rules on stable.

Reply:
```
$ cargo fmt --all
Warning: can't set `normalize_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `format_strings = false`, unstable features are only available in nightly channel.
Warning: can't set `unstable_features = true`, unstable features are only available in nightly channel.
Warning: can't set `error_on_line_overflow = false`, unstable features are only available in nightly channel.
Warning: can't set `report_todo = Always`, unstable features are only available in nightly channel.
Warning: can't set `report_fixme = Always`, unstable features are only available in nightly channel.
Warning: can't set `normalize_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `format_strings = false`, unstable features are only available in nightly channel.
Warning: can't set `unstable_features = true`, unstable features are only available in nightly channel.
Warning: can't set `error_on_line_overflow = false`, unstable features are only available in nightly channel.
Warning: can't set `report_todo = Always`, unstable features are only available in nightly channel.
Warning: can't set `report_fixme = Always`, unstable features are only available in nightly channel.
Warning: can't set `normalize_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `format_strings = false`, unstable features are only available in nightly channel.
Warning: can't set `unstable_features = true`, unstable features are only available in nightly channel.
Warning: can't set `error_on_line_overflow = false`, unstable features are only available in nightly channel.
Warning: can't set `report_todo = Always`, unstable features are only available in nightly channel.
Warning: can't set `report_fixme = Always`, unstable features are only available in nightly channel.
```

How about using default formatting rules which is available in stable channel?

Reply:
Yes, but I will fix this in another PR.